var/spack/repos/builtin/packages/py-torchgeo/package.py

```diff
@@ -16,6 +16,7 @@ class PyTorchgeo(PythonPackage):
     maintainers("adamjstewart", "calebrob6")
```

Comment:
@nilsleh you can also add yourself as a maintainer if you want. You'll get automatically notified on issues/PRs involving TorchGeo (there aren't many).